Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -68,6 +74,12 @@ private static class NativeMethods
             [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_unregister_progress_notifier", CallingConvention = CallingConvention.Cdecl)]
             public static extern void unregister_progress_notifier(SessionHandle session, ulong token, out NativeException ex);
 
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_register_connection_change_callback", CallingConvention = CallingConvention.Cdecl)]
+            public static extern ulong register_connection_change_callback(SessionHandle session, IntPtr state_change_callback_handle, out NativeException ex);
```

Comment:
Instead of passing a handle to the action action here, we can pass a handle to the managed session.

Reply:
That's applicable with Nikola's suggestion above.